### PR TITLE
Clarify expectations for Blackjack exercise

### PR DESCRIPTION
**Pull request by wmleidy**
In my opinion, this updated text more accurately reflects the expectations for candidates completing the Blackjack exercise.

### CODE REVIEW

**Comment by jgill** [comment]
I think this looks good. However, I have a question, If a senior/mid submits a solution that is only interactive in nature does that disqualify them from the position?

File: README.markdown [hunk 1]
```diff
@@ -34,25 +34,23 @@ test framework.)
...
+ * As a Player I can bust (lose immediately) when I am getting cards
  * As a Dealer I can draw cards after the player until I win or lose
+ * At the end of the round (after the Dealer's turn), the result of the round is determined
 
 **Rules:**
 
  * Bust - Occurs when all possible hand values are greater than 21 points
  * Blackjack - Occurs when a player or dealer is dealt an ace and a 10-point card
  * Dealer - Stays on 17 or above
+ * Round Flow - Player acts first, then the dealer acts if necessary
```
Comment (mullaney) [inline README.markdown]: I think Round Flow belongs with the acceptance criteria and not the **Rules**.

**Comment by wmleidy** [comment]
@jgill I did not mean to imply that interactive games are wrong or incorrect. In fact, I personally think it should be the default implementation. If you thought that's the case, then I think we need to tweak the text.

File: README.markdown (hunk 1 above)
Comment (wmleidy) [reply README.markdown]: I put this here for a few reasons:

1) There is already an implied round flow in the current use cases
2) I did not want to imply that a `Round` class should be implemented by adding a criterion for this
3) I do think round flow is part of the rules of blackjack. For example, vingt-et-un, which is similar to blackjack, has a different round flow.

**Pull request merged**